apps/condo/domains/ticket/components/TicketChanges/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import { TicketChange as TicketChangeSchema } from '../../utils/clientSchema'
+import { TicketChange } from './TicketChange'
+import { Col, Row, Typography, Button } from 'antd'
+import { useIntl } from '@core/next/intl'
+import { colors } from '@condo/domains/common/constants/style'
+// TODO(antonal): fix "Module not found: Can't resolve '@condo/schema'"
+// import { SortTicketChangesBy } from '@condo/schema'
+
+interface ITicketChangesProps {
+    ticketId: string
+}
+
+const CHANGES_PER_CHUNK = 5
+
+export const TicketChanges: React.FC<ITicketChangesProps> = ({ ticketId }) => {
+    const intl = useIntl()
+    const [displayCount, setDisplayCount] = useState(CHANGES_PER_CHUNK)
+    const TicketChangesMessage = intl.formatMessage({ id: 'pages.condo.ticket.title.TicketChanges' })
+    const FetchMoreMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.fetchMore' })
+    // TODO(antonal): get rid of separate GraphQL query for TicketChanges
+    const { objs: changes, count: total, error } = TicketChangeSchema.useObjects({
+        where: { ticket: { id: ticketId } },
+        // TODO(antonal): fix "Module not found: Can't resolve '@condo/schema'"
+        // sortBy: [SortTicketChangesBy.CreatedAtDesc],
+        // @ts-ignore
+        sortBy: ['createdAt_DESC'],
+    })
+
+    return !error && changes && changes.length > 0 && (
+        <Col span={24} style={{ marginTop: '20px' }}>
+            <Row gutter={[0, 24]}>
+                <Col span={24}>
+                    <Typography.Title level={5}>{TicketChangesMessage}</Typography.Title>
+                </Col>
+                <Col span={24}>
+                    {changes.slice(0, displayCount).map(change => (
+                        <TicketChange
+                            key={change.id}
+                            ticketChange={change}
+                        />
+                    ))}
+                    {displayCount <= total && (
+                        <Button
+                            type="text"
+                            onClick={() => {
+                                setDisplayCount(displayCount + CHANGES_PER_CHUNK)
+                            }}
+                            style={{
+                                fontSize: '16px',
+                                padding: 0,
+                                color: colors.linkColor,
+                            }}
+                        >
+                            ↓ {FetchMoreMessage.replace('{count}', Math.min(total - displayCount, CHANGES_PER_CHUNK))}
```

Comment:
use FormattedMessage component which u can import from 'react-intl'

Reply:
Used in `5ad04471`